documentation/2.0/content/userguide/tools/model_help.md

```diff
@@ -52,16 +52,16 @@ You can use the information above to construct this model section:
 ```yaml
 resources:
     JDBCSystemResource:
-        CompatibilityName: 'myName'
-        DeploymentOrder: 5
-        Target: 'ms1,ms2'
         'JDBC-1':
-            # JdbcSystemResource attributes and folders
-        SubDeployment:
-            deployment1:
-                # SubDeployment attributes and folders
-            deployment2:
-                # SubDeployment attributes and folders
+            # JDBCSystemResource attributes and folders
+            CompatibilityName: 'myName'
```

Comment:
Should we start removing quotes around strings that are no longer required from our doc samples?